types/echarts/options/series/sankey.d.ts

```diff
@@ -51,6 +51,15 @@ declare namespace echarts {
              */
             id?: string;
 
+            /**
+             * Series name used for displaying in
+             * [tooltip](https://echarts.apache.org/en/option.html#tooltip)
+             *
+             *
+             * @see https://echarts.apache.org/en/option.html#series-sankey.name
+             */
+            name?: string;
```

Comment:
I didn't see `name` under series. If you mean the name under `data`, it shouldn't be placed here. https://echarts.apache.org/en/option.html#series-sankey.data.name .
I checked with the demo but it doesn't seem to work with tooltip.